src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3161,6 +3174,17 @@ private bool CanSetNumberFormat(object parameter)
             return true;
         }
 
+        private void SetDefaultScaleFactor()
+        {
+            var defaultWorkspace = Workspaces.FirstOrDefault();
```

Comment:
it seems a lot safer to just pass in a workspace so it's explicit what scaling you are setting.

Reply:
Modified so the SetDefaultScaleFactor will receive it as parameter, check the next PR: https://github.com/DynamoDS/Dynamo/pull/13734